### PR TITLE
Update theming.rst

### PR DESCRIPTION
**Pull request by ColeLRogers**
[LEARNER-6111](https://openedx.atlassian.net/browse/LEARNER-6111)

### CODE REVIEW

**Comment by bbaker6225** [comment]
```reStructuredText
 .. warning::
 	 
     Course certificates have not yet been implemented.
```
This should be updated

**Comment by bbaker6225** [comment]
```reStructuredText
The ``locale`` directory should contain the compiled translation (.mo) files (produced by running the
``django-admin.py compilemessages`` command), and should be structured according to the conventions described in `How
Django discovers translations`_.
```

We no longer use the `compilemessages` function, since we have the translation management added in `Makefile`.  Most of the internationalization section seems unnecessary to me, except for the last paragraph "Translations included...".  I would be ok with either cutting out most of this section, or just updating the translation instructions to match what's in the new i18n docs.

File: docs/theming.rst [hunk 1]
```diff
@@ -101,6 +101,38 @@ Course certificates can be overridden at the following levels:
 * Course (e.g. edX+DemoX)
 * Course run (e.g. course-v1:edX+DemoX+4T2017) + seat type
 
+Records Theming
```
Comment (bbaker6225) [inline docs/theming.rst]: This section looks fine to me, but I really don't know much about theming.  Might be useful to get an additional review from someone who knows more about the work you did here

**Comment by mikix** [comment]
Wait, is the compilemessage stuff obsolete? It's talking about how to bundle the strings you introduce in your theme, right?

**Comment by ColeLRogers** [comment]
Added back the section with the updated command and with a link to the new Internationalization doc.

File: docs/theming.rst [hunk 2]
```diff
@@ -101,20 +101,55 @@ Course certificates can be overridden at the following levels:
...
 
 Translations included with the theme application are available to the including application by default. Any conflicts
 between translations provided by the theme application and the including application are resolved according to the
 precedence rules described in `How Django discovers translations`_.
 
+For more information on translations, look at the Internationalization_ documentation.
+
 .. _Django internationalization documentation: https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#internationalization-in-template-code
 .. _How Django discovers translations: https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#how-django-discovers-translations
+.. _Internationalization: https://github.com/edx/credentials/blob/master/docs/internationalization.rst
```
Comment (bbaker6225) [inline docs/theming.rst]: I would change this link to https://edx-credentials.readthedocs.io/en/latest/internationalization.html

**Pull request merged**